### ISSUE TITLE
Show real subscribers for double opt-in campaigns

### ISSUE THREAD

**Issue by NicoHood** [knadh/listmonk#616]
**Is your feature request related to a problem? Please describe.**
A lot of spam bots are entering my email newsletter, but those mails are never reachable. But the list overview shows all subscribers, including bots (that did not click the confimation link).

**Describe the solution you'd like**
I'd like to filter that number only to those that have double-opted in.
![image](https://user-images.githubusercontent.com/6888294/144848101-fb2ace42-da23-496b-9209-cfd2eb589f8f.png)

**Comment by NicoHood** [comment]
@knadh What is your opinion on that? Will this be considered for the next release? I guess this might be quite simple to implement?

**Comment by NicoHood** [comment]
Actually I noticed, that I opened this issue twice. Seems to be important for me :'D
https://github.com/knadh/listmonk/issues/559

**Comment by knadh** [comment]
Yep, this is good to have. It'll have to be exposed as a new field in the API, so that the total count and the confirmed count are both available. UI can be adjusted accordingly.

**Comment by knadh** [comment]
WIP

![image](https://user-images.githubusercontent.com/547147/152208747-dc42dd8b-c512-4992-9c29-578c97450624.png)
